modules/drivers/component-upi-driver/.gitignore

```
@@ -1 +1,2 @@
 /target
+/src/.envrc
```

this looks wrong, please verify. It should be on
```suggestion
/.envrc
```